backend/.eslintrc

```diff
@@ -9,5 +9,13 @@
     "global-require": "off",
     "no-underscore-dangle": "off",
     "sort-keys": ["error", "asc"]
-  }
+  },
+  "overrides": [
+    {
+      "files": ["lib/models/**/*.js"],
```

Comment:
ah ok might be a good idea, wouldn't it still be better if the schema fields are ordered? 
But just the indexes aren't or do they have to be sorted on the fields also? Otherwise could be more clear to have an inline comment and only at the part where it matters...

Reply:
Decided to put it there because someone might "autofix" something in the model and affect the  database indexing. ESlint become that tool "that" you run "that" command to make it happy. Someone could be writing an index, tried to commit but ESlint complaints, so they run `npm run lint` to fix it and them commit with the wrong index order.

That's what I had in mind when decided to put that config in there, I can change it back if you want!

Reply:
What do you think of it? Should I change it back?